eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  Directory: 'not-specified'
+
+steps:
```

Comment:
To help with debugging and seeing exactly what command line was ran please convert this to a single script call similar to https://github.com/Azure/azure-sdk-tools/blob/458a4b4113f009c999f6ed006ea2cdbb386d8332/eng/common/pipelines/templates/steps/create-pull-request.yml#L38.